fury/utils.py

```diff
@@ -138,6 +138,40 @@ def numpy_to_vtk_cells(data, is_coords=True):
     return cell_array
 
 
+def vtk_vertices_from_actor(actor):
+    """Access to vtk vertices from actor.
+
+    Parameters
+    ----------
+    actor : actor
+
+    Returns
+    -------
+    vertices : vtkarray
+
+    """
+    return actor.GetMapper().GetInput().GetPoints().GetData()
+
+
+def vtk_array_from_actor(actor, array_name):
+    """Access vtk array from actor which uses polydata.
+
+    Parameters
+    ----------
+    actor : actor
+
+    Returns
+    -------
+    output : vtkarray
+
+    """
+    vtk_array = \
+        actor.GetMapper().GetInput().GetPointData().GetArray(array_name)
+    if vtk_array is None:
+        return None
+    return vtk_array
```

Comment:
to remove, this function already exists in the same file (even if it needs to be renamed)

Look at [line 1027](https://github.com/fury-gl/fury/blob/master/fury/utils.py#L1027).

if you just want the vtk type, you can add an option in this function.

Reply:
Fixed